tests/queries/tests.py

```diff
@@ -2032,6 +2032,36 @@ def test_exists(self):
         self.assertNotIn(id, qstr)
         self.assertNotIn(name, qstr)
 
+    def test_ticket_24141(self):
```

Comment:
`tests/queries/tests.py` is already too big, please add a new file, e.g. `tests/queries/test_contains.py`:

```python
class ContainsTests(TestCase):
    def test_basic(self):
        ...
```